tutorial/tutorial-template.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+{Before using this template, read the accompanying [tutorial template guide](tutorial-template-guide)}.
+
+## Overview
+
+In this tutorial, you'll learn how to {insert brief description of the main tutorial task}. This tutorial is intended for {audience}. It assumes you have basic knowledge of:
+
+* Concept 1
+* Concept 2
+* Concept 3...
+
+By the end of this tutorial, you'll be able to:
+
+* Learning objective 1
+* Learning objective 2
+* Learning objective 3...
+
+## Background
+
+{This section is optional. Feel free to use some of the text below to help you get started.}
+
+* {product} is a {product type} that you can use to {common use case}... 
+* {product} provides many of the same features as {competitors}, but with {feature}, you can...
+* Using {feature} enables you to {pain point}...
+
+## Before you begin 
+
+{Use this section to tell users about any steps they need to complete before they start the tutorial.}
+
+Before you start the tutorial, complete these steps:
```

Comment:
Suggestion: As above, prerequisites can be more than following steps. (My suggested change could likely be improved upon.)
```suggestion
Before you start the tutorial, you should:
```